src/lib.rs

```diff
@@ -1,251 +1,252 @@
-                    )
-                    .with_system(
-                        bevy_mod_raycast::update_raycast::<PickingRaycastSet>
-                            .label(PickingSystem::UpdateRaycast)
-                            .before(PickingSystem::UpdateIntersections),
-                    )
-                    .with_system(
-                        bevy_mod_raycast::update_intersections::<PickingRaycastSet>
-                            .label(PickingSystem::UpdateIntersections),
-                    ),
-            );
-    }
-}
-
-pub struct InteractablePickingPlugin;
-impl Plugin for InteractablePickingPlugin {
-    fn build(&self, app: &mut App) {
-        app.init_resource::<PausedForBlockers>()
-            .add_event::<PickingEvent>()
-            .add_system_set_to_stage(
-                CoreStage::First,
-                SystemSet::new()
-                    .with_run_criteria(|state: Res<PickingPluginsState>| {
-                        simple_criteria(state.enable_interacting)
-                    })
-                    .with_system(
-                        pause_for_picking_blockers
-                            .label(PickingSystem::PauseForBlockers)
-                            .after(PickingSystem::UpdateIntersections),
-                    )
-                    .with_system(
-                        mesh_focus
-                            .label(PickingSystem::Focus)
-                            .after(PickingSystem::PauseForBlockers),
-                    )
-                    .with_system(
-                        mesh_selection
-                            .label(PickingSystem::Selection)
-                            .after(PickingSystem::Focus),
-                    )
-                    .with_system(
-                        mesh_events_system
-                            .label(PickingSystem::Events)
-                            .after(PickingSystem::Selection),
-                    ),
-            );
-    }
+/// Makes an entity pickable.
+#[derive(Bundle, Default)]
+pub struct PickableBundle {
+    /// The entity's configurable [`PickLayer`](focus::PickLayer)
+    pub pick_layer: focus::PickLayer,
+    /// Tracks entity [`Interaction`] state.
+    pub interaction: Interaction,
+    /// The entity's configurable [`FocusPolicy`]
+    pub focus_policy: FocusPolicy,
+    #[cfg(feature = "selection")]
+    /// Tracks entity [`PickSelection`](selection::PickSelection) state.
+    pub selection: selection::PickSelection,
+    #[cfg(feature = "highlight")]
+    /// Tracks entity [`PickHighlight`](highlight::PickHighlight) state.
+    pub highlight: highlight::PickHighlight,
 }
 
-pub struct HighlightablePickingPlugins;
-impl PluginGroup for HighlightablePickingPlugins {
-    fn build(&mut self, group: &mut PluginGroupBuilder) {
-        group.add(CustomHighlightPlugin(StandardMaterialHighlight));
-        group.add(CustomHighlightPlugin(ColorMaterialHighlight));
+/// Components needed to build a pointer
```

Comment:
This should mention that multiple pointers can be active at once, and that each one is an entity.